### DISCUSSION TITLE
How to delete generated positions from the database settings?

### DISCUSSION THREAD

**Discussion by Dying-Ducks** [akiver/cs-demo-manager#668] [Q&A] [answered]
To use the 2D Viewer the progamm needs to "generate positions" and it recommends "to delete them from the database settings when you don't need them anymore".

I noticed that my PostgreSQL folder on my disk getting bigger, but I couldn't find HOW to "delete them from the database settings". 

I would be nice if this could be added to the FAQ in the documentation.
Thank you.

**Comment by akiver** [comment]
You have to go to `Settings` -> `Database` and click the button **Optimize database**.

<img width="1058" alt="Screenshot 2023-12-06 at 09 34 42" src="https://github.com/akiver/cs-demo-manager/assets/1842524/cd0fc67e-6a1f-42cf-ab95-721b750ca42b">

**Reply by akiver** [reply, marked as answer]
A [documentation](https://cs-demo-manager.com/docs/guides/database#optimizing-the-database) is now available.

Thanks for reporting this!
